webapp/lib/app/nook/view.dart

```diff
@@ -1003,21 +1005,47 @@ class ConversationListPanelView {
     _conversationList = new LazyListViewModel(conversationListElement, onAddItemCallback: _onAddConversation);
     conversationListPanel.append(conversationListElement);
 
-    var panelFilters = new DivElement()
+    _panelFilters = new DivElement()
       ..classes.add('conversation-list-filters');
-    conversationListPanel.append(panelFilters);
+    conversationListPanel.append(_panelFilters);
 
-    conversationIdFilter = new ConversationIdFilter();
-    panelFilters.append(conversationIdFilter.conversationFilter);
+    var collapsePanelFilters = new DivElement()
+      ..append(SpanElement()..className = "fas fa-chevron-circle-down")
+      ..className = "conversation-filters-toggle"
+      ..onClick.listen((_) { _view.appController.command(UIAction.toggleFiltersPanel, ToggleFiltersPanelData(false)); });
```

Comment:
nit - use syntax for 1-liner function:
```suggestion
      ..onClick.listen((_) => _view.appController.command(UIAction.toggleFiltersPanel, ToggleFiltersPanelData(false)));
```